.expeditor/verify.pipeline.yml

```diff
@@ -6,45 +6,73 @@ expeditor:
 
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+- label: run-lint-ruby-2.4
+  command:
+    - apt-get update
```

Comment:
You can remove the `apt-get update` and `apt-get install` commands for the lint test.